docs/topics/templates/document-attributes.adoc

```diff
@@ -61,7 +61,7 @@
 // :ProductVersion: 4.0
 :ProductDistributionVersion: 5.1.Final
```

Comment:
We need to update the version to 5.1.4 here as well , right ?

Reply:
I'm not sure. The change to the installation instructions does not make use of the attribute.

:ProductDistributionVersion: is part of the next attribute in the list, :ProductDistribution: mta-cli-{ProductDistributionVersion} 

But this attribute only appears in the file about-home-var.adoc, which is part of the Rules Development Guide. The quote there is "This guide uses the `<MTA_HOME>` replaceable variable to denote the path to your {ProductShortName} installation. The installation directory is the `{ProductDistribution}` directory where you extracted the {ProductShortName} `.zip` file." 

which looks this when rendered: x-special/nautilus-clipboard
copy
file:///home/rhoch/Pictures/Rules%20Development%20Guide.png

Reply:
I doubt very much that the 5.1.Final has anything to do with the installation directory. More likely it refers to the version of the file in various installation instructions. You were looking at the wrong attribute.

That looks like a file or image tag, not a directory.

Reply:
@apinnick -- I looked at the correct attribute but apparently I was not clear enough. 

The attribute that @sshveta referred to, :ProductDistributionVersion:, is called only once in the MTA repo, in the next attribute in the list, :ProductDistribution: mta-cli-{ProductDistributionVersion}.

:ProductDistribution: itself is only called once --  in the file _about-home-var.adoc_, which is part of the Rules Development Guide. The quote there is "This guide uses the <MTA_HOME> replaceable variable to denote the path to your {ProductShortName} installation. The installation directory is the {ProductDistribution} directory where you extracted the {ProductShortName} .zip file."

So :ProductDistributionVersion: potentially affects the information in about-home-var.adoc but I don't think the change to the Maven distribution covered in the PR has any effect on :ProductDistributionVersion:, so I think we should leave :ProductDistributionVersion: alone. Since you're much more familiar with the installation instructions and information than i am, I wanted to know what you thought.